python/tests/test_factions.py

```diff
@@ -0,0 +1,225 @@
+import pandas as pd
+import pytest
+
+# Import the functions from the correct package path
+from open_discourse.helper.constants import ADDITIONAL_FACTIONS, FACTION_ABBREVIATIONS
+
+
+# In order to test the functions without triggering numpy/pandas version compatibility errors,
```

Comment:
why should that be a problem?? The tests use the same numpy and pandas versions as the main scripts, so we should in fact use all the libraries that we rely on??